pkg/cmd/scan/output/console_test.go

```diff
@@ -108,6 +108,12 @@ func TestConsole_Write(t *testing.T) {
 			args:       args{analysis: fakeAnalysisWithGithubEnumerationError()},
 			wantErr:    false,
 		},
+		{
+			name:       "test console output without deep mode",
+			goldenfile: "output_without_deep.txt",
+			args:       args{analysis: fakeAnalysisWithoutDeep()},
```

Comment:
Why don't just use `fakeAnalysis(analyser.AnalyzerOptions{Deep: false})` ?

Reply:
`fakeAnalysis` returns an instance, the test helpers returns a pointer !

Reply:
We want to have a minimal analysis for this case, `fakeAnalysis` returns an analysis with a lot of useless stuff not related to this case.

Reply:
We need an unmanaged resource with Deep: false to see the effect of the PR.